bash_completion.d/000_bash_completion_compat.bash

```diff
@@ -235,4 +235,18 @@ _init_completion()
 # shellcheck disable=SC2154  # defined in the main "bash_completion"
 _backup_glob=$_comp_backup_glob
 
+# @deprecated Use "_comp_command_offset" instead.  It should be noted that the
+# new interface "_comp_command_offset" is changed to receive an index in
+# "words" instead of that in "COMP_WORDS" as "_command_offset" did.
+_command_offset()
+{
+    # This compatibility function assumes the first argument to be the index in
+    # "COMP_WORDS" instead of that in "words".  To tell it to the new interface
+    # "_comp_command_offset", we locally make the variable "words" in the unset
+    # state.
```

Comment:
```suggestion
    # We unset the shell variable `words` locally to tell
    # `_comp_command_offset` that the index is intended to be that in
    # `COMP_WORDS` instead of `words`.
```